client/app/assets/less/inc/ant-variables.less

```diff
@@ -1,8 +1,38 @@
 /* --------------------------------------------------------
     Colors
 -----------------------------------------------------------*/
+@white:                                 #ffffff;
+@black:                                 #000000;
+@blue:                                  #2196F3;
+@red:                                   #F44336;
+@purple:                                #9C27B0;
+@deeppurple:                            #673AB7;
+@lightblue:                             #03A9F4;
+@cyan:                                  #00BCD4;
+@teal:                                  #009688;
+@green:                                 #4CAF50;
+@lightgreen:                            #8BC34A;
+@lime:                                  #CDDC39;
+@yellow:                                #FFEB3B;
+@yellow-darker:                         #fbd208;
+@amber:                                 #FFC107;
+@orange:                                #FF9800;
+@deeporange:                            #FF5722;
+@gray:                                  #9E9E9E;
+@bluegray:                              #607D8B;
+@indigo:                                #3F51B5;
+@pink:                                  #E91E63;
+@brown:                                 #795548;
+@light-gray:                            #FCFCFC;
+@gray-light:                            #828282;
+@ace:                                   #f8f8f8;
+
 @primary-color:                        #2196F3;
 
+@redash-gray: rgba(102, 136, 153, 1);
+@redash-orange: rgba(255, 120, 100, 1);
+@redash-black: rgba(0, 0, 0, 1);
+@redash-yellow: rgba(252, 252, 161, 0.75);
```

Comment:
Not important nor a blocker but perhaps this list could be minimized to the bare minimum - `@redash-gray` and `@lightblue`? Doesn't look like the other colors are being called.

Reply:
Removed